cluster/config-defaults.yaml

```diff
@@ -426,7 +426,7 @@ spotio_ocean_controller_memory: "512Mi"
 kubernetes_event_logger_enabled: "true"
 
 # enable/disable routegroup support for stackset
-stackset_routegroup_support_enabled: "false"
+stackset_routegroup_support_enabled: "true"
```

Comment:
note: This is currently set to `false` in cluster registry and thus will not be enabled by this merge.